### DISCUSSION TITLE
Call to undefined function Livewire\Concerns\data_forget(), when use $this->reset('anyVariable');

### DISCUSSION THREAD

**Discussion by deividfirewall** [livewire/livewire#6881] [7. Bugs (archived)]
### Livewire version

Livewire V3.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

When I use `$this->reset('name','phone');` in a **user_component**, show this error: _Call to undefined function Livewire\Concerns\data_forget()_
but if use `$this->reset('name','phone')` in a **user_form**, this is correct, 

### Additional information

This error ocurr only in production on a ubuntu server

In my laptop running in localhost WSL2 work correctly `$this->reset()` both (component and form). 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by jozeflambrecht** [comment]
update laravel to at least version 10.15
